doc/manpage-newsbeuter.txt

```diff
@@ -41,6 +41,9 @@ OPTIONS
 -i opmlfile, --import-from-opml=opmlfile::
        Import an OPML file
 
+-H homedir, --homedir homedir::
```

Comment:
Please use `--homedir=homedir` format to be consistent with the rest of the manpage.